src/engine/strat_engine/pool.rs

```diff
@@ -360,12 +359,12 @@ impl StratPool {
 
 impl Pool for StratPool {
     fn create_filesystems<'a, 'b>(&'a mut self,
-                                  specs: &[&'b str])
+                                  specs: &[(&'b str, Option<Sectors>)])
```

Comment:
Please update ```Pool::create_filesystems()``` header comment to indicate that the size of the last filesystem with the same name wins.